### ISSUE TITLE
0.1.8 to 0.1.9 regression: closed </p>

### ISSUE THREAD

**Issue by glensc** [xemlock/htmlpurifier-html5#31]
- https://github.com/eventum/eventum/pull/536
- https://travis-ci.org/eventum/eventum/jobs/524840361

the bump closes `</p>` tags before `<a>` and that's not valid change

```diff
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'<p>I successfully installed the <a href="https://github.com/thephpleague/commonmark-ext-autolink">https://github.com/thephpleague/commonmark-ext-autolink</a> extension!</p>'
+'<p>I successfully installed the </p><a href="https://github.com/thephpleague/commonmark-ext-autolink">https://github.com/thephpleague/commonmark-ext-autolink</a><p> extension!</p>'
/home/travis/build/eventum/eventum/tests/MarkdownTest.php:51

Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'<!--https://github.com/cebe/markdown/issues/157#issuecomment-385439965--><p>here is a <a href="http://github.com">linkref</a>.<br>and <a href="http://google.com">inline</a></p>'
+'<!--https://github.com/cebe/markdown/issues/157#issuecomment-385439965--><p>here is a </p><a href="http://github.com">linkref</a><p>.<br>and </p><a href="http://google.com">inline</a>'
```

**Comment by xemlock** [comment]
@glensc Thanks for reporting this. I will try to provide a fix today.

**Comment by xemlock** [comment]
@glensc Fixed in v0.1.10
